tests/server.test.js

```diff
@@ -53,4 +53,12 @@ describe('Signup', () => {
       .expect('Content-Type', 'text/plain')
       .expect('missing name')
   })
+  it('should respond with 200 if valid sign-up data is received', async () => {
```

Comment:
This test description seems wrong… looks like we’re expecting a 400 if the name is missing
```suggestion
  it('should respond with 400 if field are missing', async () => {
```